qiskit/execute_function.py

```diff
@@ -275,6 +302,182 @@ def execute(
 
             job = execute(qc, backend, shots=4321)
     """
+    if basis_gates is not None:
+        warnings.warn(
+            "The basis_gates argument is deprecated as of Qiskit Terra 0.21.0, "
```

Comment:
```suggestion
            "The basis_gates argument is deprecated as of Qiskit Terra 0.23.0, "
```
:)

Reply:
Heh, I'll have to do a big find and replace for that :)

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7888/commits/8baf5f1950f7c703dc2dda141ccfc12bfed5c563